Fix english typos and grammar in documentation #4398 (#108834)
Azure/azure-content: markdown templates/aml-templates/template-concepts.md

---
title: CONCEPT SUCH AS Web Services in Azure Machine Learning in 59 chars or less. Include the name Azure Machine Learning. Test title here https://moz.com/learn/seo/title-tag 

> description: This string describes the article in 115 to 145 characters. Use SEO kind of action verbs here. Such as - Learning how to do this and that using customer words. This info is displayed on the search page in line with the article date stamp. If your intro para describes your article's intent, you can use it here, edited for length.

Reviewer: ```suggestion
description: This string describes the article in 115 to 145 characters. Use SEO kind of action verbs here. For example, Learning how to do this and that using customer words. This info is displayed on the search page in line with the article date stamp. If your intro paragraph describes your article's intent, you can use it here, edited for length.
```
